includes/template-tags.php

```diff
@@ -608,6 +610,119 @@ function dokan_store_category_menu( $seller_id, $title = '' ) {
+        $vendor    = dokan()->vendor->get( $seller_id );
+        $store_url = dokan_get_store_url( $seller_id );
+        if ( $vendor instanceof Vendor ) {
+            $terms = $vendor->get_vendor_used_terms_list( $seller_id, $taxonomy );
+            echo '<ul>';
+            $_chosen_attributes = get_chosen_taxonomy_attributes();
+            foreach ( $terms as $term ) {
+                $current_values = isset( $_chosen_attributes[ $taxonomy ]['terms'] ) ? $_chosen_attributes[ $taxonomy ]['terms'] : array();
+                $option_is_set  = in_array( $term->slug, $current_values, true );
+                $filter_name = 'filter_' . wc_attribute_taxonomy_slug( $taxonomy );
+                // phpcs:ignore
+                $current_filter = isset( $_GET[ $filter_name ] ) ? explode( ',', wc_clean( wp_unslash( $_GET[ $filter_name ] ) ) ) : array();
+                if ( ! in_array( $term->slug, $current_filter, true ) ) {
+                    $current_filter[] = $term->slug;
+                }
+                $link = remove_query_arg( $filter_name, $store_url );
+                // Add current filters to URL.
+                foreach ( $current_filter as $key => $value ) {
+                    // Exclude query arg for current term archive term.
+                    if ( $value === get_current_term_slug() ) {
+                        unset( $current_filter[ $key ] );
+                    }
+
+                    // Exclude self so filter can be unset on click.
+                    if ( $option_is_set && $value === $term->slug ) {
+                        unset( $current_filter[ $key ] );
+                    }
+                }
+
+                if ( ! empty( $current_filter ) ) {
+                    asort( $current_filter );
+                    $link = add_query_arg( $filter_name, implode( ',', $current_filter ), $link );
+
+                    // Add Query type Arg to URL.
+                    if ( 'or' === $query_type && ! ( 1 === count( $current_filter ) && $option_is_set ) ) {
+                        $link = add_query_arg( 'query_type_' . wc_attribute_taxonomy_slug( $taxonomy ), 'or', $link );
+                    }
+                    $link = str_replace( '%2C', ',', $link );
+                }
+
+                $checked = '';
+                if ( $option_is_set ) {
+                    $checked = 'checked';
+                }
+                echo '
+                <li class="parent-cat-wrap">
+                    <a href=' . esc_url( $link ) . '> <input style="pointer-events: none;" type="checkbox" ' . $checked . '> &nbsp;' . $term->name . ' <span style="float: right" class="count">(' . $term->count . ')</span> </a>
+                </li>';
+            }
+            echo '</ul>';
+        }
+        ?>
+    </div>
+        <?php
+    }
+
+endif;
+/**
+ * Return the currently viewed term slug.
+ *
+ * @return int
+ */
+function get_current_term_slug() {
```

Comment:
Every time you write a global function, kindly prefix the function name with `dokan_`.